DDG4/python/DDSim/Helper/Physics.py

```diff
@@ -77,6 +77,13 @@ def pdgfile( self, val ):
       raise RuntimeError( "PDGFile: %s not found" % os.path.abspath( val ) )
     self._pdgfile = os.path.abspath( val )
 
+  @property
+  def decays(self):
+    """If true, add decay processes for all particles, only enable when creating a new physics list."""
```

Comment:
Maybe add here a bit more explanation what `new physics list` means.

Reply:
`a physics list not based on an existing Geant4 list`?

Reply:
perfect